doc/python_api_reference_vDev.md

```diff
@@ -249,11 +249,11 @@ API references for stable versions are kept on the [stim github wiki](https://gi
     - [`stim.TableauIterator.__iter__`](#stim.TableauIterator.__iter__)
     - [`stim.TableauIterator.__next__`](#stim.TableauIterator.__next__)
 - [`stim.TableauSimulator`](#stim.TableauSimulator)
+    - [`stim.TableauSimulator.__init__`](#stim.TableauSimulator.__init__)
     - [`stim.TableauSimulator.c_xyz`](#stim.TableauSimulator.c_xyz)
     - [`stim.TableauSimulator.c_zyx`](#stim.TableauSimulator.c_zyx)
     - [`stim.TableauSimulator.canonical_stabilizers`](#stim.TableauSimulator.canonical_stabilizers)
     - [`stim.TableauSimulator.cnot`](#stim.TableauSimulator.cnot)
-    - [`stim.TableauSimulator.copy`](#stim.TableauSimulator.copy)
```

Comment:
Oh. I completely forgot this method already existed. You *can't* rename it because it will break backwards compatibility. My mistake, urgh.

Reply:
It's ok. I made a leading change (as in "a leading question") :P

Done.